Pocket/pocket-ios
# ci(swiftlint): update rules

## Summary
Updated `.swiftlint.yml` file to include the updated rules and address issues for each rule as needed

## References 
N / A 

## Implementation Details
Enabled the following rules:
- for_where
- force_try
- attributes (included all and addressed warnings)
- orphaned_doc_comment
- optional_enum_case_matching
- overridden_super_call (not enabled on tests)

## Test Steps
Build and Run the app to verify that there are no warnings associated with the rules above

## PR Checklist:
- N/A Added Unit / UI tests
